diploma-project/pages/catalog/[id].jsx

```diff
@@ -40,25 +58,43 @@ const Catalog = ({user}) => {
 
     return (
         <div className="flex flex-col min-h-screen overflow-x-hidden">
-            <StandardLayout title="Catalog page" page_content={products ?
+            <StandardLayout title="Catalog page" craftsman={craftsman} toggleDrawerContent={toggleDrawerContent} setToggleDrawerContent={setToggleDrawerContent} page_content={products ?
```

Comment:
As this is getting more complex - it's getting harder to read with the ternary operator. Maybe consider a switch or something